internal/config/validate_test.go

```diff
@@ -241,33 +193,77 @@ func TestConfigValidate_ResultPathOptionalWithPytest(t *testing.T) {
 	c.ResultPath = ""
 	c.TestRunner = "pytest"
 
-	err := c.Validate(opts)
+	err := c.validate()
 	if err != nil {
 		t.Errorf("config.validate() error = %v", err)
 	}
 }
 
-func TestTargetTimeParsing(t *testing.T) {
```

Comment:
@gchan I've maintained all the tests in this PR except for this test, which I've removed.

The test seems to be testing the behvaiour of stdlib code (`time.ParseDuration`) so I think it's unnecessary. LMK if that's incorrect and I'm missing something.